release-notes/next.yml

```diff
@@ -33,6 +34,8 @@ releases:
         url: "https://github.com/coreos/fedora-coreos-tracker/issues/1300"
   37.20220910.1.0:
     issues:
+      - text: "tracker: Rebase onto Fedora 37"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1223"
```

Comment:
Hmm, that's not new to `next` though.

Reply:
I retroactively added it to the first release of `next` that had F37. Probably should have done that in a separate commit.

Reply:
Oh I see, it's an old entry. Gotcha.